src/app/Plans/PlansPage.tsx

```diff
@@ -1,14 +1,109 @@
-
-const PlansPage: React.FunctionComponent = () => (
-  <>
-    <PageSection>
-      <Title headingLevel="h1" size="lg">
-        Plans Page Title
-      </Title>
-    </PageSection>
-  </>
-);
+import {
+  Card,
+  PageSection,
+  CardBody,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Title,
+  Button,
+  Bullseye,
+  Spinner,
+  Tooltip,
+} from '@patternfly/react-core';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import PlansTable from './components/PlansTable';
+import { IAddPlanDisabledObjModel } from './types';
+import { Provider } from '@app/Providers/types';
+import { ProviderType } from '@app/common/constants';
+
+// TODO replace these with real state e.g. from redux
+import { MOCK_PLANS } from './mocks/plans.mock';
+import { MOCK_PROVIDERS } from '@app/Providers/mocks/providers.mock';
+const IsFetchingInitialPlans = false; // Fetching for the first time, not polling
+const migplans = MOCK_PLANS;
+const providers = MOCK_PROVIDERS;
+
+const PlansPage: React.FunctionComponent = () => {
+  const vmwareList: Provider[] = providers.filter((x) => x.spec.type === ProviderType.vsphere);
+  const cnvList: Provider[] = providers.filter((x) => x.spec.type === ProviderType.cnv);
+
+  const [isWizardOpen, toggleWizard] = React.useReducer((isWizardOpen) => !isWizardOpen, false);
+
+  let addPlanDisabledObj: IAddPlanDisabledObjModel = {
+    isAddPlanDisabled: false,
+    disabledText: 'Click to create a new migration plan',
+  };
+
+  if (vmwareList.length < 1) {
+    addPlanDisabledObj = {
+      isAddPlanDisabled: true,
+      disabledText: 'At least 1 VMware provider is required to create a plan.',
+    };
+  }
+
+  if (cnvList.length < 1) {
+    addPlanDisabledObj = {
+      isAddPlanDisabled: true,
+      disabledText: 'At least 1 OpenShift provider is required to create a plan.',
+    };
+  }
+
+  return (
+    <>
+      <PageSection variant="light">
+        <Title headingLevel="h1" size="lg">
+          Migration Plans
+        </Title>
+      </PageSection>
+      <PageSection>
+        {IsFetchingInitialPlans ? (
+          <Bullseye>
+            <EmptyState variant="large">
+              <div className="pf-c-empty-state__icon">
+                <Spinner size="xl" />
+              </div>
+              <Title headingLevel="h2">Loading...</Title>
+            </EmptyState>
+          </Bullseye>
+        ) : (
+          <Card>
+            <CardBody>
+              {!migplans ? null : migplans.length === 0 ? (
+                <EmptyState className={spacing.my_2xl}>
+                  <EmptyStateIcon icon={PlusCircleIcon} />
+                  <Title size="lg" headingLevel="h2">
+                    No migration plans
+                  </Title>
+                  <EmptyStateBody>
+                    Create a migration plan to select VMs to migrate to OpenShift virtualization.
```

Comment:
@vconzola told me should now be capitalizing the V in "OpenShift Virtualization".